### PR TITLE
Fix pow as per Daniil Slavins PR

### PR DESCRIPTION
**Pull request by hugohadfield**

### CODE REVIEW

File: clifford/_multivector.py [hunk 1]
```diff
@@ -273,7 +273,8 @@ def __pow__(self, other) -> 'MultiVector':
         other = int(round(other))
 
         if other == 0:
-            return 1
+            unit_out = self._newMV(dtype=self.value.dtype) + 1
+            return unit_out
```
Comment (eric-wieser) [inline clifford/_multivector.py]: Shorter as `self.layout.scalar` I think

**Comment by eric-wieser** [comment]
Would be good to add a test

File: clifford/test/test_clifford.py [hunk 2]
```diff
@@ -40,6 +40,14 @@ def test_pseudoscalar(self, algebra):
...
         ps2 = layout.pseudoScalar
         assert np.linalg.matrix_rank(np.column_stack((ps.value, ps2.value))) == 1
+        
+    def test_pow_0(self, algebra):
+        layout, blades = algebra
+        e1 = blades['e1']
+        ret = e1**0
+        assert type(ret) is type(e1)
+        assert ret == 1
+        assert ret.value.dtype == el.value.dtype
```
Comment (hugohadfield) [inline clifford/test/test_clifford.py]:
```suggestion
        assert ret.value.dtype == e1.value.dtype
```

**Comment by eric-wieser** [comment]
Fails CI with `clifford/test/test_clifford.py:43:1: W293 blank line contains whitespace`

**Pull request merged**